### PR TITLE
FlipActivity Bootstrap to IntroJS Tour

### PR DESCRIPTION
**Pull request by VedantSharma11**
hey @llaske 
i have changed bootstrap tour to inrojs js tour in Flipactivity ,pls review it.
Refer to issue:https://github.com/llaske/sugarizer/issues/1090

image

### CODE REVIEW

File: activities/Flip.activity/js/activity.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function runactivity(act,doc,colors,env,datastore,sizepalette,tutorial){
 	function init(){
 		canvas = document.getElementById('actualcanvas');
 		canvas.width = window.innerWidth;
-		canvas.height = window.innerHeight;
+		canvas.height = window.innerHeight-180;
```
Comment (llaske) [inline activities/Flip.activity/js/activity.js:34]: You should not change the canvas size.

File: activities/Flip.activity/js/activity.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ function runactivity(act,doc,colors,env,datastore,sizepalette,tutorial){
 		window.addEventListener('resize', resizeCanvas, false);
 		function resizeCanvas() {
 			canvas.width = window.innerWidth;
-			canvas.height = window.innerHeight-(+document.getElementById("main-toolbar").style.opacity ? 55 : 0);
+			canvas.height = window.innerHeight-180;
```
Comment (llaske) [inline activities/Flip.activity/js/activity.js:54]: You should not change the canvas size.

**Review by llaske — CHANGES_REQUESTED** [review]

File: activities/Flip.activity/js/activity.js (hunk 2 above)
Comment (VedantSharma11) [reply activities/Flip.activity/js/activity.js:54]: @llaske i'll correct it right away!

**Comment by VedantSharma11** [comment]
@llaske please review it i have made the changes

**Comment by llaske** [comment]
@VedantSharma11 only a part of it

**Comment by VedantSharma11** [comment]
@llaske fixed it 
but actually i had  changed the canvas size because when the 2nd popup shows up the canvas moves a bit upwards so to fix it i decreased the height which fixed the issue.

pls review

**Comment by llaske** [comment]
Yes, you're right the move should be fixed.

image

But changing the way the activity work is not a good idea. You should explore other ways. May be using some introjs events.

**Comment by VedantSharma11** [comment]
@llaske Sure i'll try some other way to fix it instead of changing the canvas size

**Comment by VedantSharma11** [comment]
@llaske i have fixed it please review

**Comment by VedantSharma11** [comment]
@llaske i have fixed it please review

**Comment by llaske** [comment]
Good @VedantSharma11 . Thanks.

**Pull request merged**